controller/controller.go

```diff
@@ -391,3 +402,27 @@ func (c *Controller) Identity() identity.Identity {
 func (c *Controller) GetEventDispatcher() event.Dispatcher {
 	return c.eventDispatcher
 }
+
```

Comment:
I would have this take the raft.Configuration directly and work with that. That way you don't have to worry about the change not having propagated yet.

Reply:
Ah, I see you're storing it and then retrieving it, so you don't have to worry about syncing state, but since the only place is used is in this method, I still think it's cleaner to pass it through.